astrobase/lcmath.py

```diff
@@ -1371,6 +1377,8 @@ def phase_bin_magseries_with_errs(phases, mags, errs,
     finite_phases = phases[finiteind]
     finite_mags = mags[finiteind]
     finite_errs = errs[finiteind]
+    if len(weights) > 10:
+        finite_weights = weights[finiteind]
 
```

Comment:
add finite_weights = None for the else statement here because we check it below.

Reply:
... yes, oops! Thanks.